OrbitLinuxTracing/PerfEventRecords.h

```diff
@@ -139,6 +139,17 @@ struct __attribute__((__packed__)) perf_event_lost {
   perf_event_sample_id_tid_time_streamid_cpu sample_id;
 };
 
+// This structure represents mmap event up to filename
+struct __attribute__((__packed__)) perf_event_mmap_up_to_filename {
```

Comment:
This is not "up to filename" as `filename` is excluded, rather "up to pgoff" (or at least it's ambiguous). Also adjust the comment.

Reply:
done.